site/content/docs/user/rootless.md

```diff
@@ -6,11 +6,12 @@ menu:
     identifier: "rootless"
     weight: 3
 ---
-Starting with kind 0.11.0, [Rootless Docker](https://docs.docker.com/go/rootless/) and [Rootless Podman](https://github.com/containers/podman/blob/master/docs/tutorials/rootless_tutorial.md) can be used as the node provider of kind.
+Starting with kind 0.11.0, [Rootless Docker](https://docs.docker.com/go/rootless/), [Rootless Podman](https://github.com/containers/podman/blob/master/docs/tutorials/rootless_tutorial.md) and [Rootless Nerdctl](https://github.com/containerd/nerdctl/blob/main/docs/rootless.md) can be used as the node provider of kind.
 
 ## Provider requirements
 - Docker: 20.10 or later
 - Podman: 3.0 or later
+- Nerdctl: 1.7 or later
```

Comment:
nit: maybe s/Nerdctl/nerdctl/g

Reply:
Thanks, it has been updated.
And add some docs for `KIND_EXPERIMENTAL_PROVIDER` : https://github.com/kubernetes-sigs/kind/pull/3532/files#diff-de2961bf197d22d2e444cd3f30d82bbc479b53b49d99edcdec48045a2aae4c26